instaloader/__main__.py

```diff
@@ -22,6 +23,15 @@
     bc3_library = False
 
 
+class ExitCodes(IntEnum):
```

Comment:
It is mainly a matter of preference, and I'm open-minded, but I have the feeling it is a little more common to name enum types in the singular (i.e. ExitCode here) than in the plural. The Python enum examples (https://docs.python.org/3/library/enum.html) also do it this way.

Reply:
I believe this is also a convention in other languages, and it does read better, especially when it's the return type of a function or when one of the values is used.